src/main/java/com/amazonaws/services/kinesis/leases/interfaces/ILeaseSelector.java

```diff
@@ -0,0 +1,30 @@
+package com.amazonaws.services.kinesis.leases.interfaces;
+
+import com.amazonaws.services.kinesis.leases.impl.Lease;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * ILeaseSelector abstracts away the lease selection logic from the application code that's using leasing.
+ * It owns filtering of the leases to be taken.
+ */
+public interface ILeaseSelector<T extends Lease> {
```

Comment:
Could you rename this to `LeaseSelector` instead of `ILeaseSelector`

Reply:
All other interfaces in this class have their names starting with 'I'.

Reply:
Yeah, that was done earlier. Going forward we are trying to avoid that.